plugins/check_ups.c

```diff
@@ -233,24 +230,26 @@ int main(int argc, char **argv) {
 		}
 
 		if (config.check_variable == UPS_UTILITY) {
-			if (config.check_crit &&
-				ups_utility_deviation >= config.critical_value) {
+			if (config.check_crit && ups_utility_deviation >= config.critical_value) {
 				result = STATE_CRITICAL;
-			} else if (config.check_warn &&
-					   ups_utility_deviation >= config.warning_value) {
+			} else if (config.check_warn && ups_utility_deviation >= config.warning_value) {
 				result = max_state(result, STATE_WARNING);
 			}
-			xasprintf(&data, "%s",
-					  perfdata("voltage", (long)(1000 * ups_utility_voltage),
-							   "mV", config.check_warn,
+			xasprintf(&data,
+					  "%s",
```

Comment:
looks better, except these lines with the format on a single line

Reply:
hm, i will play a bit more with the binpack parameters

Reply:
so, changed `BinPack`. Arguments are again more compressed. Maybe a little bit too much, but to change that the line length should be reduced I guess.
@sni: How do you like it?